cmd/manager/main.go

```diff
@@ -62,7 +62,12 @@ import (
 	//+kubebuilder:scaffold:imports
 )
 
-var scheme = runtime.NewScheme()
+var (
+	GitCommit = "undefined"
+	Version   = "undefined"
+
+	scheme = runtime.NewScheme()
+)
```

Comment:
Do we have to duplicate the var definition between here and manager-hub?

Reply:
We could have those variables in a common package, but the ldflags would then look like `-X internal.cmd.Version=1.0.0`. Would you prefer that? IMO the variables should be related to the binary in an obvious way - I am concerned that this change would make things slightly more confusing.

Reply:
How about we set `main.Version` in manager and manger-hub to `internal.cmdutils.Version`? This way we remove the dup but doesn't change the interface in the Makefile?

Reply:
So we still do `-X main.Version=1.0.0` in the ldflags, and then in the code that initializes `internal.cmdutils.Version` as well? I don't see the point of doing that. @yevgeny-shnaidman WDYT?

Reply:
> So we still do -X main.Version=1.0.0 in the ldflags

Yes.

I mean instead of having `Version = "undefined` in the manger, we set `Version = cmdutils.Version` and set `Version = "undefined"` in cmdutils.Version.